tests/Commands/MakeTraitCommandTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Tests\Commands;
+
+test('can make trait', function () {
+    $this->artisan('beyond:make:trait Users/HasActivationCodeTrait');
+
+    expect(base_path() . '/src/Domain/Users/Traits/HasActivationCodeTrait.php')
+        ->toBeFile()
+        ->toPlaceholdersBeReplaced();
+});
+
+test('can make support trait', function () {
+    $this->artisan('beyond:make:trait HasActivationCodeTrait --support');
+
+    expect(base_path() . '/src/Support/Packages/Laravel/Traits/HasActivationCodeTrait.php')
+        ->toBeFile()
+        ->toPlaceholdersBeReplaced();
+});
+
+test('namespace is correct', function () {
```

Comment:
We provide a function for that, please see https://github.com/regnerisch/laravel-beyond/blob/main/tests/Commands/MakeBuilderCommandTest.php as reference.